src/context/errors.test.ts

```diff
@@ -26,4 +27,21 @@ describe('errors', () => {
       )
     })
   })
+  describe('given composed with data', () => {
+    let result: ReturnType<typeof response>
+
+    beforeAll(() => {
+      result = response(
+        data({ name: 'msw' }),
+        errors([{ message: 'is great' }]),
+      )
+    })
+
+    it('should have body set to the given JSON nested in the "data" property', () => {
+      expect(result).toHaveProperty(
+        'body',
+        `{\"errors\":[{\"message\":\"is great\"}],\"data\":{\"name\":\"msw\"}}`,
```

Comment:
This is what we're after in the end. I tested it both in the `data` and `errors` context as both support it and do strictly are not concerned how it is achieved underneath them.